core/workspace_svg.js

```diff
@@ -211,6 +211,7 @@ Blockly.WorkspaceSvg.prototype.createDom = function(opt_backgroundClass) {
     this.addFlyout_();
   }
   this.updateGridPattern_();
+  this.recordDeleteAreas();
```

Comment:
I'm surprised we need this one since I think the delete areas (e.g. flyout, trash, etc.) won't have been positioned yet when this is called? But doesn't seem harmful.